.gitignore

```diff
@@ -113,3 +113,4 @@ venv.bak/
 
 # Scratch File
 gender_novels/scratch.py
+gender_novels/tutorials/dataviz_matplotlib.ipynb
```

Comment:
Unfortunately, this won't work--once a file has been added to version control, adding it to .gitignore doesn't take it off version control (we had tried that with scratch.py). Can you please revert this change?